tomviz/Utilities.cxx

```diff
@@ -948,5 +950,46 @@ bool jsonToFile(const QJsonDocument& document)
   return true;
 }
 
+bool moleculeToFile(vtkMolecule* molecule)
```

Comment:
We should make some time to break up utilities, it is becoming overly generic at this stage. I would suggest MoleculeUtilities, ImageDataUtilities at some point to avoid the kitchen sink. We can probably live with this for now.